### PR TITLE
Sass mixins for text

### PR DESCRIPTION
**Pull request by zachjjohns**
[Trello Ticket Link](https://trello.com/c/lRtFRgBR/239-cfc-design-system-sass-mixins-for-text-styling)

- Added _mixins.scss with the requested text styles
- Updated font-weight variables towards bottom of _variables.scss

### CODE REVIEW

File: scss/custom/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,69 @@
+@mixin heading1() {
+  font-family: $font-family-base;
+  font-size: 48px;
```
Comment (hanna2895) [inline scss/custom/_mixins.scss:3]: can you please make variables for the font sizes values as well? you can use something like xs, sm, md, lg, xl, etc. and then add them here! 

Something like: `$font-size-md: 20px`

**Review by hanna2895 — APPROVED** [review]
This looks great Zach! Thanks so much for picking this up! 

One note - I changed the branch from `master` to `main` since we made that change when we set up the netlify CMS. I'll update this in the github repo as well but just wanted to flag. 

This is ok to merge as is, but I also dropped a comment below on the font-size variables if you want to create those variables as well.

File: scss/custom/_mixins.scss (hunk 1 above)
Comment (zachjjohns) [reply scss/custom/_mixins.scss]: Done! `$font-size-md` and similar were already variables used thru Bootstrap, so I removed the first hyphens to make it like: `$fontsize-md`

**Comment by hanna2895** [comment]
Looks great, thanks Zach! Feel free to merge whenever you're ready :)

**Pull request merged**